### PR TITLE
First draft of audio/tts guide

### PR DESCRIPTION
**Pull request by mattgarrish**
Committing the first draft so we can start getting additional feedback and to make it easier to contribute edits.

I've put the guide in a new reports directory under the a11y task force's directory. If there's another place for these guides, let me know. /cc @mteixeira-wwn @Jeffxz

### CODE REVIEW

**Comment by GeorgeKerscher** [comment]
I think it looks great. IMO ready to publish and get wider review.

File: TaskForces/a11y/reports/audio-playback/index.html [hunk 1]
```diff
@@ -119,52 +122,57 @@ <h2>Full Audio Publications</h2>
 
 			<p>Some reading devices will omit the text, making it appear as though an EPUB is an audiobook, but, unlike
 				audiobooks, there is always a minimal amout of text that has to be provided. Publications with full
```
Comment (MyDK) [inline TaskForces/a11y/reports/audio-playback/index.html:124]: a tiny misspelling: amout > amount

File: TaskForces/a11y/reports/audio-playback/index.html [hunk 2]
```diff
@@ -119,52 +122,57 @@ <h2>Full Audio Publications</h2>
...
 
-			<p>When a user enables text-to-speech playback, the text content of the publication is fed to an underlying
+			<p>When a user enables Text-To-Speech playback, the text content of the publication is fed to an underlying
 				TTS engine that voices each word. For most general language, the rendering returned is reasonably good,
 				but the engines will struggle with works that contain complex terms, jargon, uncommon names, etc.
 				Readers consequently also typically have the option to have individual words spelled out, both to help
-				disambiguate similar-sounding words and also to make sense of complex words and homonyms (words spelled
-				the same but with different pronunciations) that are mispronounced.</p>
+				disambiguate similar-sounding words and also to make sense of complex words and heteronyms (words
+				spelled the same but with different pronunciations) that are mispronounced. This ability explore the
```
Comment (MyDK) [inline TaskForces/a11y/reports/audio-playback/index.html:161]: should there be a 'to' in this sentence: "This ability explore the spelling of words" > "This ability to explore the spelling of words"

File: TaskForces/a11y/reports/audio-playback/index.html [hunk 3]
```diff
@@ -119,52 +122,57 @@ <h2>Full Audio Publications</h2>
...
-				allow publishers to provide the proper phonetic pronunciation of complex words and homonyms.
+				allow publishers to provide the proper phonetic pronunciation of complex words and heteronyms.
 				Unfortunately, support for these technologies is not yet widespread.</p>
 
-			<p>So although text-to-speech synthesis provides a reasonably alternative to audio playback formats, it is
-				remains an imperfect means of reading publications.</p>
+			<p>Text-To-Speech synthesis provides a reasonable alternative to audio playback formats, and can be speeded
+				up to high rates. In addition, Text-To-Speech can also be synchronized with refreshable braille
+				displays. However, Text-To-Speech remains an imperfect means of reading publications, especially when
+				compared to the wonderful talented narraters that produce audio books.</p>
```
Comment (MyDK) [inline TaskForces/a11y/reports/audio-playback/index.html:175]: tiny misspelling: narraters > narrators

File: TaskForces/a11y/reports/audio-playback/index.html (hunk 1 above)
Comment (mattgarrish) [reply TaskForces/a11y/reports/audio-playback/index.html]: Thanks for the reports! I've corrected these typos in the latest commit.

**Pull request merged**